src/components/sbb-pearl-chain-item/sbb-pearl-chain-item.custom.d.ts

```diff
@@ -0,0 +1,18 @@
+export type LineType = 'dotted' | 'standard' | 'thin';
+
+export type DotType = 'standard' | 'thick-bullet' | 'thin-bullet' | 'double-bullet';
+
+export type Color = 'red' | 'gray' | 'black' | 'sky';
+
+export type DotSize = 'small' | 'medium' | 'large';
+
+export interface PearlChainItemAttributes {
```

Comment:
In my opinion these values should be a separate prop each. But can also be done / discussed if we later refactor the component.

Reply:
We would consider that in a later refactor.